tests/e2e/hci.go

```diff
@@ -75,11 +87,22 @@ var _ bool = ginkgo.Describe("hci", func() {
 				GetAndExpectStringEnvVar(envRemoteHCIDsUrl))
 		}
 		if vmknic4VsanDown {
-			ginkgo.By("enable vsan network on the host's vmknic in cluster4")
-			err := nicMgr.SelectVnic(ctx, "vsan", GetAndExpectStringEnvVar(envVmknic4Vsan))
+			ginkgo.By("Enable vsan network on the host's vmknic in remote cluster")
+			err := nicMgr.SelectVnic(ctx, vsanLabel, GetAndExpectStringEnvVar(envVmknic4Vsan))
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			vmknic4VsanDown = false
 		}
+		if isHostInMaintenanceMode {
+			ginkgo.By("Exit host from maintenance mode in remote cluster")
+			exitHostMM(ctx, targetHostSystem, mmStateChangeTimeout)
+			isHostInMaintenanceMode = false
+		}
+		if isTargetHostPoweredOff {
+			ginkgo.By("Power on the host used in step 3")
+			err := vMPowerMgmt(tbinfo.user, tbinfo.location, tbinfo.podname, targetHostSystemName, true)
```

Comment:
change it to camel case tbinfo --> tbInfo

Reply:
As this is a global variable and is used in many files, not changing it right now